winpr/libwinpr/utils/stream.c

```diff
@@ -96,13 +98,23 @@ wStream* Stream_New(BYTE* buffer, size_t size)
 
 	s->pool = NULL;
 	s->count = 0;
-
+	s->isStatic = FALSE;
```

Comment:
Might be a good idea to distinguish between static buffer and static `wStream`

Reply:
@awakecoding proposed some modifications from wayk now where there is a `isOwner` flag, that allows to resize statically allocated streams by doing a copy of the buffer on the heap.